javatests/arcs/showcase/imports/ImportsTest.kt

```diff
@@ -0,0 +1,36 @@
+package arcs.showcase.imports
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import arcs.core.host.toRegistration
+import arcs.showcase.ShowcaseEnvironment
+import arcs.showcase.imports.particles.AcceptImports
+import arcs.showcase.imports.particles.IngestDock
+import arcs.showcase.imports.recipes.GatherImportsPlan
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.runBlocking
+import kotlinx.coroutines.withTimeout
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@ExperimentalCoroutinesApi
```

Comment:
Might be good to add some KDoc here explaining in some more detail what the point of the test is, since it's not super obvious.

Reply:
Just noticed you have a README, so that might be good enough.